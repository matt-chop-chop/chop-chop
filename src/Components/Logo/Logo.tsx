import { Image } from "@chakra-ui/react";
import { NavLink } from "react-router-dom";
import { Link } from "@chakra-ui/next-js";
import React from "react";

const Logo = () => {
  return (
    <Link as={NavLink} href="/" width="fit-content">
      <Image
        alt="Chop Chop logo"
        height="80px"
        src="/images/logo.png"
        transition="transform 250ms ease-out"
        _hover={{
          transition: "transform 250ms ease-in",
          transform: "scale(1.1)",
        }}
      />
    </Link>
  );
};

export default Logo;
