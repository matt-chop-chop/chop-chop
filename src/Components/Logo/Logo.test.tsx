import { shallow } from "enzyme";
import React from "react";
import Logo from "./Logo";

describe("Logo", () => {
  it("is exported from Logo", () => {
    expect(Logo).toBeDefined();
  });

  it("renders as expected ", () => {
    const wrapper = shallow(<Logo />);

    expect(wrapper).toMatchSnapshot();
  });
});
