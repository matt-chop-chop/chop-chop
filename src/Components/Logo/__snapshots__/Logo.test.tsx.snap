// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders as expected  1`] = `
<ForwardRef(Link2)
  as={
    {
      "$$typeof": Symbol(react.forward_ref),
      "render": [Function],
    }
  }
  href="/"
  width="fit-content"
>
  <Image
    _hover={
      {
        "transform": "scale(1.1)",
        "transition": "transform 250ms ease-in",
      }
    }
    alt="Chop Chop logo"
    height="80px"
    src="/images/logo.png"
    transition="transform 250ms ease-out"
  />
</ForwardRef(Link2)>
`;
